modules/labs-react/package.json

```diff
@@ -41,7 +41,7 @@
     "workday"
   ],
   "peerDependencies": {
-    "react": "^17.0 || ^18.0"
+    "react": ">=16.14"
```

Comment:
If we want to throw an npm error on React 19.x (whenever that comes out), we'd make this:
```
"react": "16.14 - 18"
```

I think it would be better to be permissive in case it works fine, rather than restrictive in case it is not.